doc.js

```diff
@@ -365,21 +365,21 @@ define('doc', ['event'], function(event) {
 			},
 
 			'checked' : function(value) {
-				if(!this.attr('type').match('(checkbox|radio)')) {
-					throw Error("This element is not checkbox or radio type");
-				}
-
-				if(value === null || value === undefined) {
-					return this.first().checked;
+				if(value !== undefined && typeof value !== 'boolean') {
+					throw Error("Parameter type is not valid");
 				}
 
-				if(typeof value !== 'boolean') {
-					throw Error("Parameter type is not valid");
+				if(typeof value === 'boolean') {
+					this.each( function (el) {
+						if(!el.getAttribute('type') || !el.getAttribute('type').match('(checkbox|radio)')) {
+							throw Error("This element is not checkbox or radio");
+						}
+						el.checked = value;
+					});
```

Comment:
Parece que voltaram alguns problemas de espaçamento que o Mariel tinha apontado num code review anterior...

```suggestion
					this.each(function (el) {
						if (!el.getAttribute('type') || !el.getAttribute('type').match('(checkbox|radio)')) {
							throw Error("This element is not checkbox or radio");
						}
						el.checked = value;
					});
```

Reply:
:hammer_and_pick: